store/uistate/features/timesheet/dashboard/index.ts

```
@@ -3,12 +3,109 @@
+  startDate: string;
+  endDate: string;
+  setStartDate: (startDate: string) => void;
+  setEndDate: (endDate: string) => void;
+  userIdOnLeave: string;
+  setUserIdOnLeave: (userIdOnLeave: string) => void;
+  departmentOnLeave: string;
+  setDepartmentOnLeave: (departmentOnLeave: string) => void;
+  departmentOnLeaveGraph: string;
+  setDepartmentOnLeaveGraph: (departmentOnLeaveGraph: string) => void;
+  departmentOnAttendanceReport: string;
+  setDepartmentOnAttendanceReport: (departmentOnAttendanceReport: string) => void;
```

ESLint

[error] Replace `departmentOnAttendanceReport:·string` with `⏎····departmentOnAttendanceReport:·string,⏎··`

(prettier/prettier)